app/assets/stylesheets/pregnancies.scss

```diff
@@ -76,6 +76,7 @@ $font-size: 17px;
     text-transform: uppercase;
     letter-spacing: 1.5px;
     background: #f1f1f1;
+    padding-left: 0px !important;
```

Comment:
I'd love if we could avoid using `important` tags here, since that sets us up for potential future problems. Does this work without it?

Reply:
Yes unfortunately it requires the important.  I did not dig in any further as to what is conflicting with the precedence.

Reply:
Looks like this is duking it out with `.table > thead > tr > th` in bootstrap. Probably not winning that battle so let's merge as is.